sourcegraph/sourcegraph
# Code Insights FE: Split hasAvailable and getNonFrozenInsightCount methods

Closes https://github.com/sourcegraph/sourcegraph/issues/33883

## Test plan
- Make sure that redirection works properly if you go to `/insights`
- Make sure that creation UI submit button has proper state disable or active based on amount of non locked/frozen insights





## App preview:

- [Link](https://sg-web-vk-fix-get-started-page.onrender.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
